manifests/init.pp

```diff
@@ -82,6 +82,10 @@
   $renew_cron_hour                       = $letsencrypt::params::renew_cron_hour,
   $renew_cron_minute                     = $letsencrypt::params::renew_cron_minute,
   $renew_cron_monthday                   = $letsencrypt::params::renew_cron_monthday,
+  $systemd_date_spec                     = $letsencrypt::params::systemd_date_spec,
```

Comment:
To stay with same naming method, like with cron, can you rename this parameter `renew_systemd_datespec` ?